src/pages/index.tsx

```diff
@@ -1,18 +1,75 @@
 import * as React from 'react';
 import { NextPage } from 'next';
-import { Box, Text, Heading, Link } from '@chakra-ui/core';
+import { Box, Stack, Text, Link } from '@chakra-ui/core';
+import AnimatedIntro from '../components/AnimatedIntro';
+import Header from '../components/Header';
+import ReactIcon from '~/icons/React';
+import VueIcon from '~/icons/Vue';
+import SvelteIcon from '~/icons/Svelte';
+import AngularIcon from '~/icons/Angular';
 
 const IndexPage: NextPage = () => {
   return (
-    <Box p="4">
-      <Heading>Frontend Indonesia</Heading>
-      <Text mb="4">⚠ This website is under construction</Text>
-      <Text>
-        In the meantime, come join our{' '}
-        <Link color="teal.500" href="https://s.feid.dev/discord" isExternal>
-          Discord
-        </Link>
-      </Text>
+    <Box
+      height="100vh"
+      justifyContent="space-between"
+      margin="auto"
+      maxW="4xl"
+      display="flex"
+      flexDirection="column"
+      p={4}
+    >
+      <Header />
+      <Box as="main">
+        <AnimatedIntro />
+        <Stack spacing={10} alignItems="center">
+          <Text fontSize="2xl" verticalAlign="middle">
+            Frontend ID adalah sebuah komunitas teknologi untuk para pengembang antar muka di
```

Comment:
```suggestion
            Frontend Indonesia adalah sebuah komunitas teknologi untuk para pengembang antar muka di
```

Frontend ID -> Frontend Indonesia